src/lib/patternfly-4-cockpit.scss

```diff
@@ -0,0 +1,14 @@
+/* Here we need to import patternfly-base.scss but the fonts file from PF4 is containing
```

Comment:
Let's add a comment where the original source of that file is (or rather, "is going to be :grin: ), and that it should be kept in sync with cockpit. Same for fonts.scss and any other synced file.

Reply:
I added the commnents and links but of course these won't be available until cockpit is adjusted to this idea as well.